### PR TITLE
feat(foundations/font): add a parameter to the import-font-families()…

### LINKED ISSUE #695: feat(fonts): to split import italic fonts and the others
# 🚀 Pattern enhancement proposal

### What kind of enhancement are you proposing ?
	
- [ ] On sketch file
- [x] On Documentation
- [x] On source code

### Description

It will be possible to review the mixin of fonts import in order to import only the "regular" fonts without the "italic" fonts ?
Maybe with params inside this mixin ?

### Justification
For web performance and font file size, after a test in production, we need to be able to choose imports

### PR DESCRIPTION
**Pull request by tiloyi**
… mixin

Resolves #695

## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [x] Yes
- [ ] No

## Describe the changes

GitHub issue number or Jira issue URL: Resolves [ISSUE-695](https://github.com/adeo/mozaic-design-system/issues/695)

### CODE REVIEW

File: src/docs/Foundations/Typography/FontFamilies/code.mdx [hunk 1]
```diff
@@ -3,67 +3,143 @@ title: 'Code'
...
+
+// you will get the following code in your css:
+.your-selector {
+  font-family: 'LeroyMerlin', sans-serif;
+}
+```
+
+### The set-font-weight() mixin
+
+The `set-font-weight()` mixin allows you to set the font weight you wish to use in your selector.
```
Comment (nherwegh) [inline src/docs/Foundations/Typography/FontFamilies/code.mdx:84]: <s>wish</s> ➡️ want

**Review by nherwegh — CHANGES_REQUESTED** [review]

**Review by Marine-Fabien — APPROVED** [review]

File: src/docs/Foundations/Typography/FontFamilies/code.mdx [hunk 2]
```diff
@@ -3,67 +3,143 @@ title: 'Code'
...
+// import font families
+@include import-font-families();
+```
+
+<Highlight theme="tips" title="Good to know">
+  As you can see the code above includes a call to the{' '}
+  <strong>import-font-families()</strong> mixin. <br />
+  <br />
+  We <strong>strongly recommend</strong> that you always call this mixin in your
+  code when using the components and tools supplied by Mozaic.
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/Foundations/Typography/FontFamilies/code.mdx:22]: fixed

File: src/docs/Foundations/Typography/FontFamilies/code.mdx [hunk 3]
```diff
@@ -3,67 +3,143 @@ title: 'Code'
 order: 1
 ---
 
-# Mixins :
+## Import
 
-## Import-font-families mixin:
+In order to use the font proposed in Mozaic in your code, start by importing the `_all-settings.scss` file in the following way:
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/Foundations/Typography/FontFamilies/code.mdx:8]: fixed

File: src/docs/Foundations/Typography/FontFamilies/code.mdx [hunk 4]
```diff
@@ -3,67 +3,143 @@ title: 'Code'
...
+
+Importing the `_all-settings.scss` file allows you to manage the font in your code using one of the following mixins:
+
+- `import-font-families()`
+- `set-font-family()`
+- `set-font-weight()`
+- `set-font-face()`
+- `set-font-scale()` _([see the dedicated documentation here](/Foundations/Typography/ScaleAndSizes/code/#the-font-scale-mixin-))_
+
+## Details of the mixins
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/Foundations/Typography/FontFamilies/code.mdx:33]: fixed

File: src/docs/Foundations/Typography/FontFamilies/code.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Foundations/Typography/FontFamilies/code.mdx]: fixed

File: src/docs/Foundations/Typography/FontFamilies/code.mdx [hunk 5]
```diff
@@ -3,67 +3,143 @@ title: 'Code'
...
 
-- `LeroyMerlinSans-Web-Light` : 'light', 'normal', 'font-weight: 300'
-- `LeroyMerlinSans-Web-Regular` : 'regular', 'normal', 'font-weight: 400'
-- `LeroyMerlinSans-Web-SemiBold` : 'semi-bold', 'normal', 'font-weight: 600'
-- `LeroyMerlinSans-Web-LightItalic` : 'light', 'italic', 'font-weight: 300'
-- `LeroyMerlinSans-Web-Italic` : 'regular', 'italic', 'font-weight: 400'
-- `LeroyMerlinSans-Web-SemiBoldItalic` : 'semi-bold', 'italic', 'font-weight: 600'
+### The import-font-families() mixin
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/Foundations/Typography/FontFamilies/code.mdx]: fixed

**Review by mohamedMok — APPROVED** [review]

**Pull request merged**